README.md

```diff
@@ -1,5 +1,12 @@
 # docker-zipkin
 
+[![Build Status](https://travis-ci.org/openzipkin/docker-zipkin.svg)](https://travis-ci.org/openzipkin/docker-zipkin)
+[![zipkin-base](https://quay.io/repository/openzipkin/zipkin-base/status "zipkin-base")](https://quay.io/repository/openzipkin/zipkin-base)
```

Comment:
Not sure about the container badges. Kinda useful, but not completely clear which badge is what until you mouse over them. Maybe this is good enough.